api/v1alpha1/openapi.yaml

```diff
@@ -1425,7 +1440,7 @@ paths:
                 $ref: '#/components/schemas/RenderedDeviceSpec'
         "204":
           description: No content
-          content: {}
+          content: { }
```

Comment:
Revert all of the unnecessary changes.  Your IDE probably has different settings and changed everything.

Reply:
reverted

Reply:
though modern official way of YAML Indentation for Array in Hash is one more level inside